Client/IO/ConsoleIO/Fields/ConsoleBooleanFieldElement.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Client.Exceptions;
+
+namespace Client.IO.ConsoleIO
+{
+    public class ConsoleBooleanFieldElement : ConsoleFieldElement<bool>
+    {
+        protected override void PrintFieldName()
+        {
+            Console.WriteLine(_fieldName + " (y/n): ");
+        }
+        protected override bool ValidateInput()
+        {
+            if (_input != "y" && _input != "n")
```

Comment:
Extract "y" and "n" to consts.
In case you want to change them in the future, you will only need to change in one place instead of 4